docs/handlers.rst

```diff
@@ -0,0 +1,116 @@
+Handlers patterns
+=================
+
+When you create a new Bottery project, it will contain a file called `handlers.py`.
+This file contains the rules that allow the bot to decide the response to a message,
```

Comment:
"...bot to decide which view will respond the messages"

Reply:
:+1: